CMakeLists.txt

```
@@ -38,6 +39,7 @@ option(NEOFOAM_ENABLE_CPP_CHECK "Enable cpp check static analyzer" OFF)
 option(NEOFOAM_ENABLE_CLANG_TIDY "Enable clang tidy static analyzer" OFF)
 option(NEOFOAM_ENABLE_PCH "Enable precompiled header" OFF)
 option(NEOFOAM_ENABLE_IWYU "Enable iwyu checks" OFF)
+option(NEOFOAM_ENABLE_CUDA "Enable CUDA" ON)
```

```suggestion
option(NEOFOAM_ENABLE_CUDA "Enable CUDA" OFF)
```
should be off by default, we don't assume cuda to be installed everywhere.